tests/test_main.py

```diff
@@ -24,12 +27,39 @@ def test_generate_fails_with_existing_yml():
     """Test that ensures that a second yml file isn't generated
     without the force command."""
     result = runner.invoke(main.app, ["generate"])
+    print(result.stdout)
 
     assert result.exit_code == 0
 
 
 def test_generate_force_option_creates_yml():
     """Test that ensures the force command works correctly."""
     result = runner.invoke(main.app, ["generate"])
+    print(result.stdout)
+
+    assert result.exit_code == 0
+
+
+@pytest.mark.parametrize(
+    "assignment_path,expected_checks",
+    [
+        (
+            "./tests/test_assignment",
+            [
+                "\u2714  Complete all TODOs",
+                "\u2714  Use an if statement",
+                "|=====================================|\n|Passing all GatorGrader Checks "
+                "100.0%|\n|=====================================|",
```

Comment:
These strings seem a bit brittle--for example, we may want to change the symbol for a passing check or the number of `=` characters. It might be better to check only for behaviors we really care about maintaining, such as that the descriptions listed in the `gatorgrade.yml` appear.